service/src/test/java/uk/gov/hmcts/reform/fpl/service/CommonDirectionServiceTest.java

```diff
@@ -157,6 +158,23 @@ void constructDirectionForCCD_shouldConstructDirectionFromConfigurationAsExpecte
             .build());
     }
 
+    @Test
+    void numberDirections_shouldApplyCorrectNumberingWhenDirectionsAreMarkedAsRemoved() {
+        CaseData caseData = populateCaseDataWithSomeDirectionsMarkedAsRemoved().build();
+
+        List<Element<Direction>> directions = service.combineAllDirections(caseData);
```

Comment:
this step seems odd. Why don't you just have a method that populates a list of directions directly. It worries me using a different method to the one under test. Unit tests should exist entirely separate.